includes/container/container.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Elementor\Container;
+
+use Exception;
+use ElementorDep\DI\ContainerBuilder;
+use ElementorDep\DI\Container as DIContainer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor Container.
+ *
+ * Elementor container handler class is responsible for containerization
+ * manager classes and their dependencies.
+ *
+ * @since 3.24.0
+ */
+
+class Container extends ContainerBuilder {
```

Comment:
Something feels little bit weird, the idea of the `ContainerBuilder` class is to build and create new container, and we extends it, but now this is not a builder this is the actual container.